enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -107,16 +113,16 @@ type provider struct {
 	// Each subdirectory corresponds to a created container instance.
 	containersRoot string
 
-	// Root directory where all image layer contents will be located.
+	// Root directory where cached images are stored.
 	// This directory is structured like the following:
 	//
-	// - {layersRoot}/
+	// - {imageCacheRoot}/v1/
```

Comment:
update comment to ref imageCacheVersion?

Reply:
done

Reply:
actually nvm, I did this intentionally. This comment is specifically describing the V1 layout, so I think having it be a reference is potentially misleading.